src/components/Savings/SavingsModal/index.tsx

```diff
@@ -81,13 +81,13 @@ const SavingsModal = ({
     isOpen: boolean
 }): JSX.Element => {
     const { i18n } = useLingui()
-    const { account } = useActiveWeb3React()
+    const { account, chainId } = useActiveWeb3React()
     const [balance, setBalance] = useState<string>('0')
     const [txStatus, setTxStatus] = useState<TransactionStatus>({ status: 'None' })
     const reduxDispatch = useDispatch()
     const getData = sendGa
 
-    const { g$Balance, savingsBalance } = useSavingsBalance(10, SupportedV2Networks.FUSE)
+    const { g$Balance, savingsBalance } = useSavingsBalance(10, SupportedV2Networks.CELO)
```

Comment:
see comment above

Reply:
no longer hardcoded, but just loads for every support network savings